packages/create-redwood-app/src/create-redwood-app.js

```diff
@@ -315,8 +246,109 @@ import { name, version } from '../package'
 
   const startTime = Date.now()
 
+  // Engine check Listr. Separate Listr to avoid https://github.com/cenk1cenk2/listr2/issues/296
+  // Boolean flag
+  let hasPassedEngineCheck = null
+  // Array of strings
+  let engineErrorLog = []
+  // Docs link for engine errors
+  const engineErrorDocsLink = terminalLink(
+    'Tutorial - Prerequisites',
+    'https://redwoodjs.com/docs/tutorial/chapter1/prerequisites'
+  )
+
+  await new Listr(
+    [
+      {
+        title: 'Checking node and yarn compatibility',
+        skip: () => {
+          if (yarnInstall === false) {
+            return 'Warning: skipping check on request'
+          }
+        },
+        task: () => {
+          return new Promise((resolve) => {
+            const { engines } = require(path.join(templateDir, 'package.json'))
+
+            // this checks all engine requirements, including Node.js and Yarn
+            checkNodeVersion(engines, (_error, result) => {
+              if (result.isSatisfied) {
+                hasPassedEngineCheck = true
+                return resolve()
+              }
+              const logStatements = Object.keys(result.versions)
+                .filter((name) => !result.versions[name].isSatisfied)
+                .map((name) => {
+                  const { version, wanted } = result.versions[name]
+                  return style.error(
+                    `${name} ${wanted} required, but you have ${version}`
+                  )
+                })
+              engineErrorLog = logStatements
+              hasPassedEngineCheck = false
+              return resolve()
+            })
+          })
+        },
+      },
+    ],
+    { rendererOptions: { clearOutput: true } }
+  ).run()
+
+  // Show a success message if required engines are present
+  if (hasPassedEngineCheck === true) {
+    console.log(style.success(`✔️ Compatability checks passed.`))
+  }
+
+  // Show an error and prompt if failed engines check
+  if (hasPassedEngineCheck === false) {
+    console.log(style.error(`✖️ Compatability checks failed.`))
+    console.log(`${engineErrorLog.join('\n')}`)
+    console.log(
+      style.error(
+        `\nThis might make your RedwoodJS project incompatible with some deploy targets.`
+      )
+    )
+    console.log(style.header(`\nRelated documentation:`))
+    console.log(style.warning(`${engineErrorDocsLink}\n`))
```

Comment:
Definitely don't want to bikeshed on this too much, but I thought there was a bit too much red so I'm proposing that we keep the colors to the icons and focus on keeping the structure of the output visually consistent:

<img width="917" alt="image" src="https://user-images.githubusercontent.com/32992335/200071579-c7df3063-c5d5-474d-ba4b-2c177e0cd081.png">

![image](https://user-images.githubusercontent.com/32992335/200071923-111d13a5-bd7b-4a5b-9e3b-432d9eaf7fe2.png)

(This suggestion depends on another that removes the styling in the `engineErrorLog`)

```suggestion
    console.log(
      [
        `  ${style.warning(figures.warning)} ${engineErrorLog.join('\n')}`,
        '',
        `    This may make your project incompatible with some deploy targets`,
        `    See ${engineErrorDocsLink}`,
        '',
      ].join('\n')
    )
```

(Before for reference)

<img width="917" alt="image" src="https://user-images.githubusercontent.com/32992335/200072350-a4a6539f-7294-4b66-ad00-3a86b48f7cff.png">

Reply:
I agree - I like this way better!